cardano-cli/src/Cardano/CLI/EraBased/Run/StakeAddress.hs

```diff
@@ -56,11 +58,18 @@ runStakeAddressCmds = \case
   StakeAddressDeregistrationCertificateCmd sbe stakeIdentifier mDeposit outputFp ->
     runStakeAddressDeregistrationCertificateCmd sbe stakeIdentifier mDeposit outputFp
 
-runStakeAddressKeyGenCmd :: ()
+runStakeAddressKeyGenCmd_ :: ()
   => KeyOutputFormat
   -> VerificationKeyFile Out
   -> SigningKeyFile Out
   -> ExceptT StakeAddressCmdError IO ()
+runStakeAddressKeyGenCmd_ fmt vkFp skFp = void $ runStakeAddressKeyGenCmd fmt vkFp skFp
```

Comment:
Please remove this function that is polluting the namespace while providing little value: callers can call void themselves 👍